2-Webpack/toDoListApp.js

```diff
@@ -0,0 +1,179 @@
+
+(function () {
+  let idSerializer = 0;
+  createAddTaskContainer();
+  if (localStorage){
```

Comment:
Extract all the logic that accesses the `localStorage` into a different file, separating this concern is very important and will come handy if you want to change the persistence layer